File: templates/matlab/quantum_semiconductor_score_radar.m
function fig = quantum_semiconductor_score_radar()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('radar', 3007, 'quantum and semiconductor analysis: multi-metric radar', 'quantum and semiconductor analysis', 'multi-metric radar');
end

File: templates/python/quantum_semiconductor_score_radar.py
"""quantum_semiconductor_score_radar: 量子与半导体多维评分雷达（radar 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='quantum and semiconductor analysis: multi-metric radar'):
    return make_template_figure('radar', seed=3007, title=title, domain='quantum and semiconductor analysis', topic='multi-metric radar')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)
